code/modules/mob/living/simple_animal/abnormality/aleph/nobody_is.dm

```diff
@@ -598,3 +709,14 @@
 //Objects
 /obj/effect/reflection // Hopefully temporary or at least removed someday
 	layer = ABOVE_ALL_MOB_LAYER
+
+//Allow for titana's laws to exist for nobody is I think
```

Comment:
I'd test this code thoroughly. I believe there is the possibility of titania dying due to abno_host.bullet_act and abno_host.attacked_by which could cause a slew of runtimes due to titania being deleted early.